scripts/start.sh

```diff
@@ -94,6 +94,9 @@ fi
 # Delete the modified feconf.py file(-i.bak)
 rm assets/constants.js.bak
 
+# Set the configuration for CKEDITOR which prevents addition of new line everytime save button is pressed.
```

Comment:
It's a pleasure. Will find out a way and will fix it.
@seanlip

Reply:
@seanlip I did research regarding the issue and came to the conclusion that we would have to make a lot of changes in our code in multiple files in order to implement this same code. I think we should add a single line of code to start.sh for this issue.

Reply:
@seanlip 
[This](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_configuration.html) is the official documentation CKEDITOR 4. This shows the possible ways in which configuration can be changed. Then I went through our codebase. The ways to change configuration, other than the one I had proposed, require a textarea and their exclusive instantiating function. But we use placeholders. Hence the conclusion.

Reply:
@seanlip Even if we plan to do it the other way, as you suggested, we will have to hardcode the container and it's CSS. And I don't think you would find it appropriate because it would spoil the generalized layout of our website.

Reply:
I'm sorry, but we are not making custom changes to third-party libraries in the start script. This leads to a non-obvious dependency that needs to be maintained every time the library is upgraded. A different way needs to be found.

I have looked through the page you linked to and I think it has some good suggestions that do not involve modifying the original distribution files in the CKEditor installation. I think that those are worth pursuing further.

@bansalnitish and @AllanYangZhou are the experts on CKEditor; it might be worth consulting them if you are stuck.